main.go

```diff
@@ -1,16 +1,62 @@
 package main
 
 import (
+	"flag"
 	"github.com/bowenislandsong/winc_node_setup/pkg/config"
 	"github.com/bowenislandsong/winc_node_setup/pkg/ec2_instances"
 	"log"
+	"os"
 )
 
 func main() {
-	sessAWS := config.AWSConfigSess()
-	oc, err := config.OpenShiftConfig()
-	if err != nil {
-		log.Fatalf("Failed to get client, %v", err)
+	if len(os.Args) < 2 {
+		log.Fatal("Please use one argument either 'create' or 'destroy' a node")
 	}
-	ec2_instances.CreateEC2WinC(sessAWS, oc, "", "", "")
+	createFlag := flag.NewFlagSet("create", flag.ExitOnError)
+	destroyFlag := flag.NewFlagSet("destroy", flag.ExitOnError)
+	if os.Args[1] == "create" {
+		// subflags of create
+		// openshift cluster region
+		credPath := createFlag.String("awscred", "", "Required: absolute path of aws credentials")
+		credAccount := createFlag.String("awsaccount", "openshift-dev", "account name of the aws credentials") // Default accounts for dev team in OpenShift
+		dir := createFlag.String("dir", "./", "path to 'winc-setup.json'.")
```

Comment:
Would it make more sense to be able to feed this in the same way we do for the installer that way we keep the config inche dir with the info for the deployment?

Reply:
That’s a great idea, and also what I was hoping for as well. However, there is no information telling me the path where Openshift-installer dir is at. We could argue that most likely kubeconfig is in the directory’s auth folder, but we can’t be sure. Hence here we ask as a flag to specify ( I just noticed that we might need to parse the dir than handle as a string as I am doing it now. I will change it in a bit. ) the default value is the current directory just like the installer